### PR TITLE
Use configured domain on expired cookies

### PR DESCRIPTION
**Pull request by iaroslav-ciupin**
# TL;DR
When expiring cookie during logout, we must use same configured domain as when setting those cookies.

## Type
 - [x] Bug Fix
 - [ ] Feature
 - [ ] Plugin

## Are all requirements met?

 - [ ] Code completed
 - [ ] Smoke tested
 - [x] Unit tests added
 - [ ] Code documentation added
 - [ ] Any pending items have an associated Issue

## Complete description
 _How did you fix the bug, make the feature etc. Link to any design docs etc_

## Tracking Issue
_Remove the '*fixes*' keyword if there will be multiple PRs to fix the linked issue_

fixes https://github.com/flyteorg/flyte/issues/<number>

## Follow-up issue
_NA_
OR
_https://github.com/flyteorg/flyte/issues/<number>_

### CODE REVIEW

File: auth/cookie_manager_test.go [hunk 1]
```diff
@@ -25,124 +25,84 @@ func TestCookieManager_SetTokenCookies(t *testing.T) {
...
-	cookieSetting := config.CookieSettings{
-		SameSitePolicy: config.SameSiteDefaultMode,
-		Domain:         "default",
-	}
-
-	manager, err := NewCookieManager(ctx, hashKeyEncoded, blockKeyEncoded, cookieSetting)
-	assert.NoError(t, err)
+	t.Run("logout_access_cookie", func(t *testing.T) {
+		cookie := manager.getLogoutAccessCookie()
+		assert.True(t, time.Now().After(cookie.Expires))
```
Comment (katrogan) [inline auth/cookie_manager_test.go:79]: should we check the domain is set here too?

**Comment by iaroslav-ciupin** [comment]
CI failures should be fixed in https://github.com/flyteorg/flyteadmin/pull/609

**Review by katrogan — APPROVED** [review]
